bin/storm.py

```diff
@@ -278,7 +278,7 @@ def kill(*args):
         print_usage(command="kill")
         sys.exit(2)
     exec_storm_class(
-        "org.apache.storm.command.kill_topology",
+        "org.apache.storm.command.KillTopology",
```

Comment:
you have also added wait as alternative argument name. should it be added in the command doc?